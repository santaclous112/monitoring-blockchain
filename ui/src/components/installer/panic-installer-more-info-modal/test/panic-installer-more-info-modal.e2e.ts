import { newE2EPage } from '@stencil/core/testing';

describe('panic-installer-more-info-modal', () => {
  it('renders', async () => {
    const page = await newE2EPage();
    await page.setContent('<panic-installer-more-info-modal></panic-installer-more-info-modal>');

    const element = await page.find('panic-installer-more-info-modal');
    expect(element).toMatchSnapshot();
  });
});
