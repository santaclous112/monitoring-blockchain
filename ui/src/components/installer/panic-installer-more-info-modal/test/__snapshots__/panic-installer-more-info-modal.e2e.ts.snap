// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-installer-more-info-modal renders 1`] = `
<panic-installer-more-info-modal class="hydrated">
  <svc-content-container class="hydrated">
    <!---->
    <div style="display: flex; justify-content: center;">
      <svc-button class="hydrated">
        <!---->
        <ion-button class="button button-default button-solid hydrated ion-activatable ion-focusable md" size="default">
          <ion-icon aria-label="checkmark" class="hydrated md" name="checkmark" role="img" slot="start"></ion-icon>
          Got it!
        </ion-button>
      </svc-button>
    </div>
  </svc-content-container>
</panic-installer-more-info-modal>
`;
